api/types/plugin.go

```diff
@@ -127,6 +128,12 @@ type PluginArgs struct {
 	Value []string
 }
 
+// PluginKernelModule represents a Loadable Kernel Modules (LKM) to modprobe before enabling a plugin
+type PluginKernelModule struct {
+	PluginSetting
```

Comment:
What is the need for `PluginSetting` here? It doesnt seem to be used anywhere in the code.